### DISCUSSION TITLE
How to listen to multiple ports when use worker_threads mode?

### DISCUSSION THREAD

**Discussion by snowinszu** [uNetworking/uWebSockets.js#1122] [Q&A]
In this [example](https://github.com/uNetworking/uWebSockets.js/blob/master/examples/WorkerThreads.js), how can I  listen to multiple ports?

**Comment by uNetworkingAB** [comment]
If you want to listen to multiple ports, you might as well skip that load balancing entirely and just use the fact that you have multiple ports, as a way to balance load.

**Comment by snowinszu** [comment]
Let's say we need to listen to 80 and 443 for http and https, how can I do that?

**Comment by uNetworkingAB** [comment]
You need two apps; SSLApp and APP and you need to register the same handlers on all URL routes. And you just call listen on them with their respective port

**Comment by snowinszu** [comment]
Thanks, it is working. May I ask, should I use cluster or multi_thread? Which one is more performant?

**Comment by uNetworkingAB** [comment]
They should have identical perf. 

My persoanl preferrence is WorkerThreads becaue, technically, it should be lower resource usage but in practise probably the same.
